AnnoyancesFilter/sections/popups.txt

```diff
@@ -89,6 +89,10 @@ sodapdf.com##.be-tooltip
 !
 ! Specific sites
 !
+hiroba.dpoint.jp###gtm-promotionModal-overlay
+hiroba.dpoint.jp##.js-modal-promotion-banner
+hiroba.dpoint.jp#$#body[data-modal-overlay="open"] > div#wrapper { position: static !important; }
```

Comment:
We have `general_extensions.txt` file for CSS(`#$#`) rules.
Added directly.

Reply:
also if you block popup using CSS, all related rules must be also CSS, because not all platform support CSS rules.

Reply:
I assumed they're annoyances, anyway thx for your advice, that makes sense.